tests/test_bot.py

```diff
@@ -385,6 +400,97 @@ def test(_, url, data, *args, **kwargs):
                                                switch_pm_text='switch pm',
                                                switch_pm_parameter='start_pm')
 
+    def test_answer_inline_query_current_offset_error(self, bot, inline_results):
+        with pytest.raises(ValueError, match=('`current_offset` and `next_offset`')):
+            bot.answer_inline_query(1234,
+                                    results=inline_results,
+                                    next_offset=42,
+                                    current_offset=51)
```

Comment:
```suggestion
            bot.answer_inline_query(
                1234,
                results=inline_results,
                next_offset=42,
                current_offset=51
            )
```

Reply:
again, we use that style all over the place. Either we change the style completely or not at all, IMHO